shop-service/product-service/productsService/index.ts

```diff
@@ -0,0 +1,27 @@
+import PRODUCTS_LIST from '../mockData';
+
+type ProductType = {
```

Comment:
Typescript's `type` keyword is better for creating type aliases. I would recommend to use `interface` keyword in order to create complex types, including `dto`'s

Reply:
done